samples/apps/org.wso2.carbon.uuf.sample.features-app/src/main/pages/helpers/block-helpers/each.hbs

```diff
@@ -1,108 +1,122 @@
 {{layout "org.wso2.carbon.uuf.sample.foundation.main"}}
 {{title "each | Block Helper Sample | " @config.appName}}
+{{fragment "org.wso2.carbon.uuf.sample.foundation.highlight" theme=@config.highlightjsTheme}}
+
 {{#fillZone "content"}}
-    <h3>The 'each' Block Helper.</h3>
+    <h3><b>each</b> Helper</h3>
     <hr />
     <p>You can iterate over a list using the 'each' helper. </p>
     <p>Inside the block, you can use this to reference the element being iterated over.</p>
-    <pre>
+    <figure class="highlight">
+        <pre class="code lang-handlebars">
         &lt;ul class="people_list"&gt;
-          \{{#each people}}
+            \{{#each people}}
                 &lt;li&gt;\{{this}}&lt;/li&gt;
-          \{{/each}}
-        &lt/ul&lt
-    </pre>
+            \{{/each}}
+        &lt;/ul&gt;</pre>
+    </figure>
     <p>For example: </p>
-    <pre>
+    <figure class="highlight">
+        <pre class="code lang-json">
         {
           people: [
             "Yehuda Katz",
             "Alan Johnson",
             "Charles Jolley"
           ]
-        }
-    </pre>
+        }</pre>
+    </figure>
     <p>will result in:</p>
-    <pre>
+    <figure class="highlight">
+        <pre class="code lang-handlebars">
         &lt;ul class="people_list"&gt;
-          &lt;li&gt;Yehuda Katz&lt;/li&gt;
-          &lt;li&gt;Alan Johnson&lt;/li&gt;
-          &lt;li&gt;Charles Jolley&lt;/li&gt;
-        &lt;/ul&gt;
-    </pre>
+            &lt;li&gt;Yehuda Katz&lt;/li&gt;
+            &lt;li&gt;Alan Johnson&lt;/li&gt;
+            &lt;li&gt;Charles Jolley&lt;/li&gt;
+        &lt;/ul&gt;</pre>
+    </figure>
     <h3>output:</h3>
```

Comment:
these are hardcoded values, but need to add the actual output by executing the template. can do this as an improvement